aruba/features/renew/renew-with-csr-service.feature

```diff
@@ -30,29 +33,39 @@ Feature: renew action with `-csr service` option
       And it should write private key to the file "k.pem"
       And it should write certificate to the file "c.pem"
       And it should output Pickup ID
-    When I renew the certificate in TPP using the same Pickup ID with flags -no-prompt -cert-file c1.pem -key-file k1.pem -csr service -key-password newPassw0rd!
```

Comment:
why did you modify the password value?

Reply:
I felt that one better for having in our test cases